src/modules/ToDoList/ToDoList.js

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import '../../styles/ToDoListStyles/ToDoList.css'
+
+import ToDoTask from "./ToDoTask";
+
+
+class ToDoList extends React.Component {
+    constructor(props) {
+        super(props);
+
+        console.log("props.tasks: ", this.props.tasks);
+
+        this.state = {
+            tasks: this.props.tasks,
+            inputValue: '',
+            editValue: ''
+        };
+    }
+    static getDerivedStateFromProps(props, state){
+        if(props.tasks!== state.tasks){
+            return {tasks: props.tasks };
+            // this.classMethod();
+        }
+    }
```

Comment:
this lifecycle should always return something. Other words, add `return null;`

Reply:
pay attention to the warnings that appear in your console, sometimes they can be important